paddle/operators/CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ op_library(add_op SRCS add_op.cc add_op.cu)
 op_library(mean_op SRCS mean_op.cc mean_op.cu)
 
 op_library(mul_op SRCS mul_op.cc mul_op.cu DEPS math_function)
+op_library(element_wise_mul_op SRCS element_wise_mul_op.cc element_wise_mul_op.cu DEPS math_function)
```

Comment:
Elementwise is one word.
elementwise_mul_op

Reply:
Thanks.
Done.